### PR TITLE
Zo

### PR DESCRIPTION
**Pull request by zshrestha2**

### CODE REVIEW

**Comment by cfulks73** [comment]
Don't merge - Blocked by #5 so we can avoid future headaches with cached files causing and it'll also make reviewing the PR easier.

**Review by Murazor-byte — APPROVED** [review]

File: Card.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+{
+    public enum Color{
+        Red, Black
+    }
+
+    public enum Suit{
+        Hearts, Diamonds, Spades, Clubs
+    }
+
+    public int value;
```
Comment (cfulks73) [inline Card.cs:14]: Class fields should be private w/ getters (don't add setters since the class should be immutable)

File: solitaire.cs [hunk 2]
```diff
@@ -0,0 +1,202 @@
+using System;
+using System.Collections.Generic;
+
+namespace CardGames
+{
+    public class solitaire
```
Comment (cfulks73) [inline solitaire.cs:6]: Class names should be capitalized

File: solitaire.cs [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+                        }
+                    }
+                }
+            }
+
+            return valid;
+        }
+
+        //should check if the cards are in asssednig order and are in same suits
+        private bool validSuitMove()
```
Comment (cfulks73) [inline solitaire.cs:172]: C# Method names are PascalCase (capitalize all the words).

**Review by cfulks73 — CHANGES_REQUESTED** [review]
I have a few small changes.

Also, the solitaire.cs file name should be capitalized as well.

**Review by jackelyny — APPROVED** [review]

**Pull request closed without merging**